dist/index.js

```diff
@@ -3567,9 +3568,13 @@ function run() {
                 core.setFailed(`The GitHub API for comparing the base and head commits for this ${github_1.context.eventName} event returned ${response.status}, expected 200. ` +
                     "Please submit an issue on this action's GitHub repo.");
             }
-            const regex = new RegExp(`/${filter}\\b`, 'g');
+            const regexInclude = new RegExp(include, 'g');
             // Get the changed files from the response payload.
-            const files = response.data.files.filter(file => file.filename.match(regex));
+            let files = response.data.files.filter(file => regexInclude.test(file.filename));
+            if (exclude) {
```

Comment:
I think we should add some documentation about how this work. For example, what happens when you have the same file in both include and exclude. Just so that people find this information on the README instead of needing to check the code.

Reply:
Yes you're right, I totally zapped the README update and it's indeed very much needed, I'll work on that tomorrow or Friday.